valhalla/thor/worker.h

```diff
@@ -84,6 +84,14 @@ class thor_worker_t : public service_worker_t {
   void parse_filter_attributes(const Api& request, bool is_strict_filter = false);
   sif::TravelMode mode;
   std::vector<meili::Measurement> trace;
+  std::vector<PathInfo> m_path_infos;
```

Comment:
because of this change a great deal of the diff is not really important to the overal point of this pr. can you please highlight the bit of the code that gets to the core issue it is solving?